pkg/audit/verifier.go

```diff
@@ -126,6 +126,32 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe, skip map[s
 
 	successNodes := getSuccessNodes(ctx, nodes, failedNodes, offlineNodes, containedNodes)
 
+	totalInPointer := len(stripe.Segment.GetRemote().GetRemotePieces())
```

Comment:
out of curiosity, i assume by this part in the code we know this segment isn't inline?

Reply:
Yes, the cursor specifically selects a remote stripe